src/libraries/Microsoft.PowerFx.Core/Parser/TexlParser.cs

```diff
@@ -1718,16 +1731,25 @@ private bool EatTid(TokKind tid)
             return false;
         }
 
-        // Returns the current token if it's of the given kind and moves to the next token.
-        // If the token is not the right kind, reports an error, leaves the token, and returns null.
-        private Token TokEat(TokKind tid)
+        /// <summary>
+        /// Returns the current token if it's of the given kind and moves to the next token.
+        /// If the token is not the right kind, reports an error, leaves the token, and returns null.
+        /// </summary>
+        /// <param name="tid">TokenKind.</param>
+        /// <param name="addError">Will not add token mis-match error when true.</param>

```

Comment:
>Will not add token mis-match error when true. [](http://example.com/codeflow?start=35&length=45)

The other way around - it will not add the error if the value is `false`.